reports.py

```diff
@@ -549,7 +549,25 @@ def parser_plot_coverage_common(parser=argparse.ArgumentParser()):    # parser n
         type=int,
         help="The max coverage depth (default: %(default)s)"
     )
-    parser.add_argument('-l', dest="read_length_threshold", default=None, type=int, help="Read length threshold")
+    parser.add_argument('-l',
+        dest="read_length_threshold",
+        default=None,
+        type=int,
+        help="Read length threshold"
+    )
+    parser.add_argument(
+        '--binLargePlots',
+        dest="bin_large_plots",
+        action="store_true",
+        help="Plot summary read depth in one-pixel-width bins for large plots."
+    )
+    parser.add_argument(
```

Comment:
The `--binningSummaryStatistic` argument can be constrained to valid options by passing a list to the [`choices` kwarg](https://docs.python.org/3/library/argparse.html#choices).

Reply:
Also, thanks for adding this!

Reply:
Great idea! Thanks!